ant-cli/Cargo.toml

```diff
@@ -27,7 +27,7 @@ ant-bootstrap = { path = "../ant-bootstrap", version = "0.1.4" }
 ant-build-info = { path = "../ant-build-info", version = "0.1.23" }
 ant-logging = { path = "../ant-logging", version = "0.2.45" }
 ant-protocol = { path = "../ant-protocol", version = "0.3.3" }
-autonomi = { path = "../autonomi", version = "0.3.5", features = [ "loud" ] }
```

Comment:
This feature although I agree not the most elegant way to do it, made the UX great. Removing it will result in a frustrating user experience while waiting for uploads.